### ISSUE TITLE
Can't install via gitpack

### ISSUE THREAD

**Issue by jolesh** [dominiksalvet/asus-fan-control#129]
**Environment**

- Linux Mint 20 Ulyana 64-bit
- AMD Ryzen 5 4600H with Radeon Graphics × 12

**Description**

- Can't install with gitpack

**Additional context**
Log:
`>>> running install for github.com/dominiksalvet/asus-fan-control
local repository fresh
reading status database
using automatic candidate version
installing version 3.13.0
E9: missing script for local installation on Linux
repository developer may help
`

**Comment by dominiksalvet** [comment]
It is perfectly correct behavior -- you cannot install AFC locally. You need to have root access and install it globally. Use `sudo gitpack install <afc-url>`.

Nevertheless, as you have an AMD system, AFC will not work anyway, because AMD systems are not supported.

**Comment by jolesh** [comment]
Thank you.
It is really sad that not even ASUS is supporting it, I wrote them and their "solution" was to switch to Windows.

**Comment by dominiksalvet** [comment]
> Thank you. It is really sad that not even ASUS is supporting it, I wrote them and their "solution" was to switch to Windows.

Agree. Before the existence of AFC, I tried it too. Had very similar outcomes... 😁
